pkg/controllers/dynakube/injection/conditions.go

```diff
@@ -0,0 +1,34 @@
+package injection
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	metaDataEnrichmentConditionType   = "MetadataEnrichment"
+	codeModulesInjectionConditionType = "CodeModulesInjection"
+
+	nsAndSecretsCreatedReason  = "SecretsCreated"
```

Comment:
forgot to update the variable name :D

Reply:
Left on purpose, because I want the Reason and Message to have the same prefix to indicate they belong together. 

`SecretsCreatedMessage` would not fit there - Maybe renaming the prefix to `nsMappedAndSecretsCreated`, but that's rather long.

Reply:
so my main problem with the naming is that (unfortunately) the `injection` reconciler's outcome is not as straightforward.

the `oneagent` reconciler has the `DaemonSet...` conditions as its outcomes, even though it also creates Secrets.

the `injection` reconciler has the "outcome" that "many things have been set up" 😅 
- as of https://github.com/Dynatrace/dynatrace-operator/pull/3327 you would even have `NamespacesMappedSecretsCreatedStatutsUpdated"